inception-ui-search/src/main/java/de/tudarmstadt/ukp/inception/app/ui/search/sidebar/SearchAnnotationSidebar.html

```diff
@@ -91,6 +161,13 @@ <h3 class="panel-title">
   <tr wicket:id="results" style="display: table-row;">
     <td style="text-align: center;"><a wicket:id="showSelectedDocument"><span
         wicket:id="sentence"></span></a></td>
+      <td>
+    <div class="checkbox">
```

Comment:
You cannot have divs in a table. Better create another `<td>...</td>` before the existing one and move the checkbox into it - or even simply move the checkbox into the existing `<td>`.